mne/channels/layout.py

```diff
@@ -145,6 +149,10 @@ def read_layout(kind, path=None, scale=True):
     -------
     layout : instance of Layout
         The layout.
+
+    See Also
+    --------
+    save
```

Comment:
This probably needs to be `mne.Layout.save` or whatever the class name is to be rendered properly (please check all refs render properly to the correct links)